tests/unit/nupic/algorithms/nab_detector_test.py

```diff
@@ -0,0 +1,273 @@
+                          },
+                          'timestamp_dayOfWeek': None
+                        },
+                        'verbosity': 0
+                      },
+                      'clEnable': False,
+                      'spParams': {
+                        'columnCount': 2048,
+                        'synPermInactiveDec': 0.0005,
+                        'randomSP': 0,
+                        'maxBoost': 1.0,
+                        'spatialImp': 'cpp',
+                        'inputWidth': 0,
+                        'spVerbosity': 0,
+                        'useHighTier': 0,
+                        'synPermConnected': 0.1,
+                        'synPermActiveInc': 0.0015,
+                        'seed': 1956,
+                        'numActiveColumnsPerInhArea': 40,
+                        'coincInputPoolPct': 0.8,
+                        'globalInhibition': 1
+                      },
+                      'trainSPNetOnlyIfRequested': False,
+                      'clParams': {
+                        'alpha': 0.035828933612158,
+                        'regionName': 'CLAClassifierRegion',
+                        'steps': '1',
+                        'clVerbosity': 0
+                      },
+                      'tpParams': {
+                        'columnCount': 2048,
+                        'activationThreshold': 13,
+                        'pamLength': 3,
+                        'cellsPerColumn': 32,
+                        'permanenceDec': 0.1,
+                        'minThreshold': 10,
+                        'inputWidth': 2048,
+                        'maxSynapsesPerSegment': 32,
+                        'outputType': 'normal',
+                        'initialPerm': 0.21,
+                        'globalDecay': 0.0,
+                        'maxAge': 0,
+                        'newSynapseCount': 20,
+                        'maxSegmentsPerCell': 128,
+                        'permanenceInc': 0.1,
+                        'temporalImp': 'cpp',
+                        'seed': 1960,
+                        'verbosity': 0
+                      },
+                      'anomalyParams': {
+                        'anomalyCacheRecords': None,
+                        'autoDetectThreshold': None,
+                        'autoDetectWaitRecords': 5030
+                      },
+                      'spEnable': True,
+                      'inferenceType': 'TemporalAnomaly',
+                      'tpEnable': True
+                      },
+                    'predictAheadTime': None
+                  }
+    sensorParams = modelParams["modelParams"]["sensorParams"]\
+                                ["encoders"]["value"]
+    sensorParams["resolution"] = \
+      max(0.001, (1.2 - 0.2) / sensorParams.pop("numBuckets"))
+    model = ModelFactory.create(modelParams)
+
+    self.assertIs(type(model).__name__, "CLAModel", msg="The created model is "
```

Comment:
Any reason why? [The documentation](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsInstance) says `assertIs` is more exact.